Backend/stream_mp3local.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# Usage: ./stream_mp3local.sh <WSPort> <PathToMP3>
+
+sox -tmp3 $2 -traw -e signed-integer -b 8 -c 1 -r 48000 - | ./aucmp | wscat -b -l $1
```

Comment:
Instead of making this only stream mp3s you could remove the `-tmp3` option and have sox determine the filetype automatically. That way this script can be used for other audio files as well.